src/targeting.js

```diff
@@ -374,7 +374,9 @@ export function newTargeting(auctionManager) {
       const bidsToFilter = bidsByReferrer[lastLocation] || filterBidsByAdUnit(bidsReceived);
       const allowedSizes = getAllowedSizes();
 
-      bidsToProcess = bidsToFilter.filter(bid => isBidSizeAllowed(bid, allowedSizes));
+      bidsToProcess = bidsToFilter
+        .filter(bid => isBidSizeAllowed(bid, allowedSizes))
+        .filter(bid => !(isBidCached(bid) && getBidName(bid) === 'teads'));
```

Comment:
I would set an array of blacklisted bidders in order to make it easier add new ones to the black list if we don't want to perform bid caching on them. Moreover, what do you think to gather it on a different function? Something like:

`isBidAllowed = () => !(isBidCached(bid) && blacklistedCacheBidders.includes(getBidName(bid)))`

Reply:
Besides that, we could add a test to cover it.

Reply:
Cool!